packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/types.ts

```diff
@@ -39,3 +42,12 @@ export enum CardStateType {
   LOCKED = 'LOCKED',
   TERMINATED = 'TERMINATED'
 }
+
+export type BalanceType = {
+  symbol: string
+  value: string
```

Comment:
why the value here is a string? what if you have to make some computation with it? Isn't it better to have it as a number?

Reply:
is a string because that is what I get from backend